oas_docs/output/kibana.yaml

```
@@ -51523,6 +51528,966 @@ paths:
+              discriminator:
+                propertyName: type
+              oneOf:
+                - $ref: '#/components/schemas/Synthetics_browserMonitorFields'
+                - $ref: '#/components/schemas/Synthetics_httpMonitorFields'
+                - $ref: '#/components/schemas/Synthetics_icmpMonitorFields'
+                - $ref: '#/components/schemas/Synthetics_tcpMonitorFields'
+        required: true
+      responses:
+        '200':
+          description: A successful response.
+      summary: Create a monitor
+      tags:
+        - synthetics
+  /api/synthetics/monitors/_bulk_delete:
+    post:
+      description: |
+        Delete multiple monitors by sending a list of config IDs.
+      operationId: delete-synthetic-monitors
+      requestBody:
+        content:
+          application/json:
+            examples:
+              bulkDeleteRequestExample1:
+                description: Run `POST /api/synthetics/monitors/_bulk_delete` to delete a list of monitors.
+                value: |-
+                  {
+                    "ids": [
+                      "monitor1-id",
+                      "monitor2-id"
+                    ]
+                  }
+            schema:
+              type: object
+              properties:
+                ids:
+                  description: An array of monitor IDs to delete.
+                  items:
+                    type: string
+                  type: array
+              required:
+                - ids
+        required: true
+      responses:
+        '200':
+          content:
+            application/json:
+              examples:
+                deleteMonitorsResponseExample1:
+                  description: A response from successfully deleting multiple monitors.
+                  value: |-
+                    [
+                      {
+                        "id": "monitor1-id",
+                        "deleted": true
+                      },
+                      {
+                        "id": "monitor2-id",
+                        "deleted": true
+                      }
+                    ]
+              schema:
+                items:
+                  description: The API response includes information about the deleted monitors.
+                  type: object
+                  properties:
+                    deleted:
+                      description: |
+                        If it is `true`, the monitor was successfully deleted If it is `false`, the monitor was not deleted.
+                      type: boolean
+                    ids:
+                      description: The unique identifier of the deleted monitor.
+                      type: string
+                type: array
+      summary: Delete monitors
+      tags:
+        - synthetics
+  /api/synthetics/monitors/{config_id}:
```

```suggestion
  /api/synthetics/monitors/{id}:
```

I think it makes sense to use the term `id` here as the API response for the GET route will return this value as `id`, and `config_id` is more of a developer-level saved object field name that end users usually will not need to concern themselves with.

---

Thanks! I've made a change to the synthetic_apis.yaml file in b1a3ef3b9c022e23a37720ab51e22cff797b16c4 then regenerated these output files.